waspc/examples/todoApp/todoApp.wasp

```diff
@@ -8,7 +8,14 @@ app todoApp {
   ],
   auth: {
     userEntity: User,
-    methods: [ UsernameAndPassword ],
+    // externalAuthAssociationEntity: SocialLogin,
```

Comment:
NOTE: I am disabling Google auth by default in the todo app, just so server won't crash if you don't have the Google project envars.